app/controllers/users_controller.rb

```diff
@@ -0,0 +1,35 @@
+class UsersController < ApplicationController
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def destroy
+    @user = User.find(params[:id])
+    @user.destroy
+    flash[:notice] = "Account successfully deleted!"
+    redirect_to root_path
+  end
+
+  def update
+    @user = User.find(params[:id])
+    @email = params[:user][:email]
+    if @user.email != @email
```

Comment:
rails māk izmaiņas čekot un nevajadzīgu update netaisīs, tāpēc šī vieta nav nepieciešama :)

Reply:
es removosu , bet man vienkarsi low level mentallity , nevajadzigu bloku ar kodu checkot kad tas nav vajadzigs, gruti noskatities 😅

Reply:
nevermind , kad iznemu to if , tad ja nav izmainas vins aizmet uz root page un parada pazinojumu ka aizsutits email saistiba ar email nomainu